lib/features/create/Create.js

```diff
@@ -3,7 +3,8 @@ var LOW_PRIORITY = 750;
 var MARKER_OK = 'drop-ok',
     MARKER_NOT_OK = 'drop-not-ok',
     MARKER_ATTACH = 'attach-ok',
-    MARKER_NEW_PARENT = 'new-parent';
+    MARKER_NEW_PARENT = 'new-parent',
+    MARKER_FRAME_NOT_OK = 'frame-drop-not-ok';
```

Comment:
Why do we need a separate class here? Couldn't we add a class such as `djs-frame` to `djs-shape` elements?

This way users could define specialized CSS declarations like `djs-frame.drop-not-ok` (or the like) and we'd not need to touch the `Create` and `Move` features at all.